run-integration-tests.py

```diff
@@ -392,6 +397,12 @@ def __exit__(self, tpe, value, traceback):
         default=False,
         action="store_true",
         help="Use testpypi for testing pip installation")
+    parser.add_argument(
```

Comment:
nit: add a check below that cannot be --user-localpypiartifact and --use-test-pypi?